src/Pages/Keys.razor

```diff
@@ -58,7 +58,7 @@
                         {
                             @StringHelper.TruncateHeadAndTail(context.XPUB, 6);
                             <span>&nbsp</span>
-                            <Button Color="Color.Primary" Clicked="@(() => CopyStrToClipboard(context.XPUB))" Size="Size.Small" Outline>
+                            <Button Color="Color.Primary" Clicked="@(async () => await CopyStrToClipboard(context.XPUB))" Size="Size.Small" Outline>
```

Comment:
Do you really need to change this?

The => implies a return in the line. So you're actually returning a Task.